src/style-sheets/pages/_page-not-found-view.scss

```diff
@@ -1,17 +1,110 @@
 .page-not-found-view-container {
-  position: relative;
-  width: 100%;
-  height: 80vh;
-
-  .page-not-found-view-body {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    text-align: center;
+  padding: 70px 0 10% 0;
+  font-size: 24px;
+
+  display: flex;
+  justify-content: center;
+
+  & .page-not-found-view-body {
+    width: 85%;
+
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    & svg {
+      width: 272px;
+      height: 196px;
+    }
+
+    & .page-not-found-view-text-container {
+      width: 100%;
+      margin-top: 5%;
+      text-align: center;
+
+      display: flex;
+      align-items: center;
+      flex-direction: column;
+
+      & p {
+        color: $olTLightCopy;
+        margin: 6px;
+        line-height: 1.3em;
+        font-family: $olBaseFont;
+      }
+
+      & button {
+        margin: 10px 0;
+        font-size: 1.25rem;
+        line-height: 1.56rem;
+
+        text-align: center;
+        font-weight: 600;
```

Comment:
I know the font weight in figma says 600 but it looks so much bolder than the mock up. This will be fine for now